lib/github.cue

```diff
@@ -128,4 +130,46 @@ import "strings"
 		_default_branch: body.repository.default_branch
 		_revision:       body.head_commit.id
 	}
+
+	github_pr_comment: {
+		#input:   string | *"GitHub PR comment to \(#repo)"
+		#prInput: string | *"GitHub PR to \(#repo)"
+
+		#repo:    =~"^[^/]+/[^/]+$"
+		#comment: string
+
+		_prRepo: #repo
```

Comment:
can't use `_repo`, causes pr input to refer to the `_repo` at the bottom which becomes a cyclic dependency on inputs.